firebase-appdistribution/test-app/src/main/java/com/googletest/firebase/appdistribution/testapp/ScreenshotDetectionFeedbackTrigger.kt

```diff
@@ -31,7 +31,9 @@ private constructor(private val infoTextResourceId: Int, handler: Handler) :
   ContentObserver(handler), Application.ActivityLifecycleCallbacks {
 
   private val seenImages = HashSet<Uri>()
-  private var requestPermissionLauncher: ActivityResultLauncher<String>? = null
+  // TODO(lkellogg): this is getting too complex - simplify it by, for example, only enabling this
+  //   on a per-activity basis instead of app-wide
```

Comment:
I think permissions are always app-wide. But I agree that this needs to be simplified. I think requesting the permission in the `MainActivity` (maybe when the trigger is selected) should do the trick. Let's decide who should implement this simplification.

Reply:
I meant enabling the trigger app-wide. I'll clarify.